index.html

```diff
@@ -31,13 +31,13 @@ <h1 class="name">
       </div>
       <div class="footer">
         <div class="links">
-          <a href="https://github.com/crclark96/" target="_blank">
+          <a href="https://github.com/udellug/" target="_blank">
             Github
           </a>
-          <a href="mailto:crclark96@gmail.com" target="_blank">
+          <a href="https://udellug.slack.com" target="_blank">
             Slack
           </a>
-          <a href="./resume.pdf" target="_blank">
+          <a href="./about.html" target="_blank">
```

Comment:
Can we have this point to `./about` which will go to `./about/index.html` instead so that we can have nice urls?

Reply:
Pointing to `./about` leads to a page like this:
![screenshot-Mon-07-Oct-2019-04:47:15-PM-EDT](https://user-images.githubusercontent.com/7076013/66347384-36428600-e922-11e9-824d-b5c63c080267.png)

The alternative of pointing to `./about/index.html` seems uglier to me

Reply:
That's because you're viewing it through your filesystem. In an Apache environment `./about` will look like `udellug.github.io/about` in the browser bar but will serve the contents of `udellug.github.io/about/index.html`.